client/source/utilities/TestAPI.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import * as requests from './axiosRequests';
+
+function testGetRequest() {
+  requests.get('products')
+    .then((response) => console.info(response.status, response.data))
+    .catch((error) => error);
+}
+
+function testPutRequest() {
+  requests.put('reviews/1278333/helpful', { review_id: 0 })
+    .then((response) => console.info(response.status, response.data))
```

Comment:
Thank you so much for including these tests! Super helpful examples. 

No action, just an observation. I saw this logged `204 ''` in my browser console. I think per Learn, we may want to use Status: 204 NO CONTENT. We can probably each handle this on client-side ourselves when we do a .then and .catch.

Reply:
I'm not sure what you mean by this. The status code `204` can only mean `no content`, they are one and the same.